.openapi-generator/templates/partial_model_generic.mustache

```diff
@@ -364,7 +364,7 @@
     # Calculates hash code according to all attributes.
     # @return [Integer] Hash code
     def hash
-      [{{#vars}}{{name}}{{^-last}}, {{/-last}}{{/vars}}].hash
+      [{{#vars}}{{#vendorExtensions.x-is-hash-attribute}}@{{name}}{{/vendorExtensions.x-is-hash-attribute}}{{^vendorExtensions.x-is-hash-attribute}}{{name}}{{/vendorExtensions.x-is-hash-attribute}}{{^-last}}, {{/-last}}{{/vars}}].hash
```

Comment:
@evanstoner the combination of this and the transformation.jq file update make this magic happen.. basically we ensure that instead of referencing self.hash in a recursive manner, we are using the instance attribute of `@hash`